frontend/assets/stylesheets/components/_icons.scss

```diff
@@ -115,6 +115,13 @@
     }
 }
 
+.icon-holder--small {
+    width: 20px; height: 20px;
```

Comment:
Personally not super-keen on this style, find it a bit harder to scan/parse what the code is doing. Unless it was like...

    $iconWidth: 20px;
    width: $iconWidth; height: $iconWidth;